api/views.py

```diff
@@ -50,3 +53,14 @@ def image(self, request, **kwargs):
             encoded_img = base64.b64encode(img_file.read()).decode('utf-8')
 
         return HttpResponse(encoded_img, content_type=img.content_type)
+
+
+class FollowersViewSet(viewsets.ModelViewSet):
+    renderer_classes = [JSONRenderer]
+    pagination_class = page_number_pagination_class_factory([('type', 'follower')])
+    serializer_class = FollowersSerializer
+    permission_classes = [permissions.IsAuthenticated]
+    http_method_names = ['get', 'put', 'delete']
```

Comment:
```suggestion
    http_method_names = ['get']
```
For now I'd stick with GET unless you want to add tests that verify the behavior (which I don't think is supported yet because we haven't implemented `to_internal` on the serializer)